elasticjob-cloud/elasticjob-cloud-executor/src/test/java/org/apache/shardingsphere/elasticjob/cloud/executor/prod/TaskExecutorTest.java

```diff
@@ -127,4 +130,20 @@ public void assertShutdown() {
     public void assertError() {
         taskExecutor.error(executorDriver, "");
     }
+    
+    @Test
+    @SneakyThrows
+    public void testConstructor() {
```

Comment:
Please use assertXXX instead of testXXX for test case method

Reply:
Ok